src/main/scala/NumberToLongShort.scala

```diff
@@ -0,0 +1,48 @@
+
+import scala.util.{Failure, Success, Try}
+
+class NumberToLongShort {
+
+  lazy val english: List[String] = List(" ", " thousand and ", " million ", " billion ", " trillion ", " quadrillion ", " quintillion ", " sextillion ")
+  lazy val french: List[String] = List(" ", " mille et ", " million ", " milliard ", " billion ", " billiard ", " trillion ", " triliard ")
+
+  def checkIfNumber(input: String, lang: List[String]): String = {
+    Try {
+      input.toLong
+    } match {
+      case Success(a) => langOutput(input, lang)
+      case Failure(a) => "Enter a valid positive number"
+    }
+  }
+
+  def langOutput(input: String, lang: List[String]): String = {
+    if (input.contains('-')) {
+      langNeg(input, lang)
+    }
+    else {
+      langPos(input, lang)
+    }
+  }
+
+  def langPos(input: String, lang: List[String]): String = {
```

Comment:
`langPos` and `langNeg` look similar, could be done in one line sacrificing clarity for each `def`